### PR TITLE
create simple status web page

### PR DESCRIPTION
**Pull request by tdawson**
This creates a simple status webpage.
We can make it fancier at a later date.

### CODE REVIEW

File: eln-check.py [hunk 1]
```diff
@@ -148,5 +149,33 @@ def diff_with_rawhide(package, eln_build=None, rawhide_build=None):
 
     f.close()
     s.close()
+    os.system("sort -u -o %s %s" % (args.status, args.status))
 
     logging.info("Total differences {0}".format(counter))
+
+    # Create Webpage
```
Comment (sturivny) [inline eln-check.py:159]: IMHO better to use jinja template for this

Comment (tdawson) [reply eln-check.py:159]: OK, I'll look at that.

**Comment by sturivny** [comment]
Thanks.
An example: https://realpython.com/primer-on-jinja-templating/

**Comment by tdawson** [comment]
I think you are correct.  Using jinja will allow us to not have to mess with the script, just the template, to make this a more complex webpage.
This final webpage is still simple, but this is to just get something there.

**Review by sturivny — APPROVED** [review]
LGTM

**Comment by tdawson** [comment]
This can't automaticlaly merge due to the checks failling with "Cloud does not exist: kubernetes"
Can this be merged?

**Comment by sturivny** [comment]
@msrb ^^

**Comment by tdawson** [comment]
Rebased and now the checks are passing.
But I still do not have permissions to merge.

**Pull request merged**